assayist/processor/main_analyzer.py

```diff
@@ -97,6 +97,8 @@ def run(self):
         build_type = None
         if task_info:
             build_type = task_info['method']
+        elif self.is_container_build(build_info):
+            build_type = 'buildContainer'
```

Comment:
I would be interested in koji build for container which does not have associated koji task.
Could you provide some example?  (I mean buildId)

Reply:
@lslebodn a container build doesn't have a task ID associated with, but the UI stitches this together based on data in the extra field.

Reply:
I should have checked method `is_container_build`

It already rely on the extra field
```
build_info['extra'].get('container_koji_task_id')
```

Sorry for noise and thank you for the answer.